src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -65,118 +136,1921 @@ const ActorInit En_Kaizoku_InitVars = {
 
 // static ColliderCylinderInit sCylinderInit = {
 static ColliderCylinderInit D_80B8AB7C = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON | BUMP_HOOKABLE, OCELEM_ON, },
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON | BUMP_HOOKABLE,
+        OCELEM_ON,
+    },
     { 20, 50, 0, { 0, 0, 0 } },
 };
 
 // static ColliderQuadInit sQuadInit = {
 static ColliderQuadInit D_80B8ABA8 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_PLAYER | AT_TYPE_ENEMY, AC_NONE, OC1_NONE, OC2_NONE, COLSHAPE_QUAD, },
-    { ELEMTYPE_UNK4, { 0xF7CFFFFF, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL | TOUCH_UNK7, BUMP_NONE, OCELEM_NONE, },
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_PLAYER | AT_TYPE_ENEMY,
+        AC_NONE,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_QUAD,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        { 0x00000000, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL | TOUCH_UNK7,
+        BUMP_NONE,
+        OCELEM_NONE,
+    },
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
 
-#endif
+u32 D_80B8ABF8[] = {
+    0x0600F8E4, 0x0600EF9C, 0x0600E1C8, 0x0600DBE4, 0x060058B8, 0x06004860, 0x06002730,
+    0x06001578, 0x06001390, 0x060003CC, 0x06000F5C, 0x0600E8BC, 0x0600ED1C, 0x06005644,
+    0x0600F288, 0x060043E4, 0x06003A3C, 0x06002BA0, 0x06001E9C,
+};
+
+u8 D_80B8AC44[] = {
+    0x00, 0x00, 0x02, 0x00, 0x00, 0x02, 0x02, 0x02, 0x02, 0x02,
+    0x02, 0x02, 0x00, 0x02, 0x02, 0x02, 0x02, 0x00, 0x02, 0x00,
+};
+
+Color_RGBA8 D_80B8AC58 = { 0xC8, 0xA0, 0x78, 0xFF };
+Color_RGBA8 D_80B8AC5C = { 0x82, 0x5A, 0x32, 0xFF };
+
+Vec3f D_80B8AC60 = { 0.0f, -1.5f, 0.0f };
+Vec3f D_80B8AC6C = { 0.0f, -0.2f, 0.0f };
+Vec3f D_80B8AC78 = { 300.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC84 = { 0.0f, -3000.0f, 0.0f };
+Vec3f D_80B8AC90 = { 400.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC9C = { 1600.0f, -4000.0f, 0.0f };
+Vec3f D_80B8ACA8 = { -3000.0f, -2000.0f, 1300.0f };
+Vec3f D_80B8ACB4 = { -3000.0f, -2000.0f, -1300.0f };
+Vec3f D_80B8ACC0 = { 1000.0f, 1000.0f, 0.0f };
+
+u32 D_80B8ACCC[] = {
+    0x060093B8, 0x06009978, 0x0600A0F8, 0x06009978, 0x00000000,
+};
+
+extern AnimationHeader D_060058B8;
+
+extern FlexSkeletonHeader D_0600D828;
```

Comment:
Instead of externing these, could you include the object file (in the `.h` file) and use the appropriate symbols. Note you should also be able remove the `EnKaizoku` section from `undef_syms.txt` when you are done.